MaybeGame/GameMaster.cs

```diff
@@ -3,69 +3,92 @@
+                e.DataDump();
+        }
 
-            public void DumpPlayersData()
-            {
-                foreach (var p in _players)
-                    p.DataDump();
-            }
+        public void Start()
+        {
+            WriteGameMenu();
+        }
+
+        private void WriteGameMenu()
+        {
+            Console.WriteLine("\nMenu:\n" +
+                        "1. Add new player\n" +
+                        "2. Add Enemies\n" +
+                        "3. Dump all player information\n" +
+                        "4. Dump all enemy information\n" +
+                        "5. Quit\n");
 
-            public void Start()
+            var success = int.TryParse(Console.ReadLine(), out var menu);
+            if (success == false)
             {
+                Console.WriteLine("You have entered a wrong number");
                 WriteGameMenu();
             }
 
-            private void WriteGameMenu()
+            var playerWantsToQuit = false;
+            switch (menu)
             {
-                Console.WriteLine("\nMenu:\n" +
-                            "1. Add new player\n" +
-                            "2. Dump all information\n" +
-                            "3. Quit" +
-                            "4. LevelUp");
-
-                var success = int.TryParse(Console.ReadLine(), out var menu);
-                if (success == false)
-                {
-                    Console.WriteLine("You have entered a wrong number");
-                    WriteGameMenu();
-                }
+                case 1:
+                    AddNewPlayer();
+                    break;
+                case 2:
+                    AddEnemies();
+                    break;
+                case 3:
+                    DumpPlayersData();
+                    break;
+                case 4:
+                    DumpEnemiesData();
+                    break;
+                case 5:
+                    playerWantsToQuit = true;
+                    break;
+                default:
+                    Console.WriteLine("I am sorry. but this doesn't work");
+                    break;
+            }
 
-                var playerWantsToQuit = false;
-                switch (menu)
-                {
-                    case 1:
-                        AddNewPlayer();
-                        break;
-                    case 2:
-                        DumpPlayersData();
-                        break;
-                    case 3:
-                        playerWantsToQuit = true;
-                        break;
-                    default:
-                        Console.WriteLine("I am sorry. but this doesn't work");
-                        break;
-                }
+            if (playerWantsToQuit) return;
+            else Start();
+        }
 
-                if (playerWantsToQuit) return;
-                else Start();
-            }
+        private void AddNewPlayer()
```

Comment:
Method does too much. Split into two:

1. ParsePlayerData() -> parse player data from console and return it
2. AddNewPlayer() -> uses ParsePlayerData() and adds the new player to list